.github/workflows/deploy-sdk.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         # uses: kenji-miyake/setup-sd@v1
         uses: levibostian/setup-sd@add-file-extension # Using fork until upstream Action has bug fixed in it.
 
+      - name: Generate SDK Size Report
+        run: ./gradlew generateSdkSizeReport -PsaveReportTo=reports
+
+      - name: Stage SDK Size Report
+        run: git add reports/
+
```

Comment:
I haven't been able to test this as this only runs when we prepare for release. So would appreciate more eyes on it.

About the workflow, the change is to create and store report in `main` branch whenever a release is made. [Added more details about it on Linear](https://linear.app/customerio/issue/MBL-157/track-the-binary-size-of-our-android-sdk#comment-5bc03dee).

Reply:
I would rather suggest move the `Stage SDK Size Report` part in the Semantic assets section we have, we already stage files in it.